packages/ui/src/coreModules/crud/higherOrderComponents/createEnsureAllItemsFetched.js

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable class-methods-use-this */
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { compose } from 'redux'
+import config from 'config'
+import actionCreators from '../actionCreators'
+import {
+  actionCreators as keyObjectActionCreators,
+  globalSelectors as keyObjectGlobalSelectors,
+} from '../keyObjectModule'
+
+const createEnsureAllItemsFetched = ({
+  resource,
+  relationships = ['parent'],
```

Comment:
I think the default should be no relationships. not all resources have a parent relationship

Reply:
true -> will change

Reply:
fc5acd8